src/main/java/bio/terra/service/configuration/ConfigurationService.java

```diff
@@ -226,6 +227,7 @@ private void setConfiguration() {
         addParameter(FIRESTORE_SNAPSHOT_CACHE_SIZE, appConfiguration.getFirestoreSnapshotCacheSize());
         addParameter(FIRESTORE_VALIDATE_BATCH_SIZE, appConfiguration.getFirestoreValidateBatchSize());
         addParameter(FIRESTORE_QUERY_BATCH_SIZE, appConfiguration.getFirestoreQueryBatchSize());
+        addParameter(DRS_LOOKUP_MAX, appConfiguration.getDrsLookupMax());
```

Comment:
needs to be changed to match rename
```suggestion
        addParameter(DRS_LOOKUP_MAX, appConfiguration.getMaxDrsLookups());
```